runtime/server_http_response.go

```diff
@@ -33,81 +32,61 @@ import (
 // ServerHTTPResponse struct manages request
 type ServerHTTPResponse struct {
```

Comment:
please do not make these private fields public. It allows consumers to mess with too much of the state.

Reply:
Sure, no problem.